modules/aws-outposts-requirements-limitations.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="aws-outposts-requirements-limitations_{context}"]
+= AWS Outposts on {product-title} requirements and limitations
+
+By configuring your {product-title} cluster to accommodate the requirements and limitations of this platform, you can manage the resources on your Outpost similarly to those on a cloud-based AWS cluster.
+
+* To extend an {product-title} cluster on AWS into an Outpost, you must have installed the cluster into an existing VPC.
+
+* {product-title} clusters on AWS include the `gp3-csi` and `gp2-csi` storage classes.
+These classes correspond to Amazon Elastic Block Store (EBS) gp3 and gp2 volumes.
+{product-title} clusters use the `gp3-csi` storage class by default, but AWS Outposts does not support EBS gp3 volumes.
+
```

Comment:
For the section, we'd better add those extra limitations.

- To create a volume in the Outpost instance, the CSI driver determines the Outpost ARN based on the topology keys stored on the CSINode objects. To ensure that the CSI driver uses the correct topology values, it is necessary to use the `WaitForConsumer` volume binding mode and avoid setting `allowed topologies` on any new storage class created.
- For the outposts remote workers and public workers mixed configuration, the public AWS Elastic Block volume couldn't be attached to the outposts remote workers, the outposts volumes couldn't be attached to public workers as well (the outposts volume logic persistent volume nodeAffinity already has the required selector could avoid it attach to a public worker while the public volume logic persistent volume nodeAffinity doesn't prevent it to be attach to an outposts worker).
cc to @duanwei33

Reply:
Good point - these conditions were present in the storage topic that I was asked to remove. I will add them in to the limitations topic so we have them stated :+1: